app/services/ReferenceDataService.scala

```diff
@@ -43,29 +41,29 @@ class ReferenceDataServiceImpl @Inject() (connector: ReferenceDataConnector) ext
   def getControlType(code: String)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[ControlType] = {
     lazy val default                       = ControlType(code, "")
     val queryParams: Seq[(String, String)] = Seq("data.code" -> code)
-    connector.getControlTypes(queryParams).map(_.headOption.getOrElse(default)).recover {
+    connector.getControlTypes(queryParams).map(_.head).recover {
```

Comment:
Might be prudent to not let these calls fail quietly (i.e. remove the recover) - could be indicative of some kind of reference data issue that we should be alerted by